src/site/_collections/algolia-posts.js

```diff
@@ -42,7 +42,7 @@ function limitText(fulltext, limit = 7500) {
 module.exports = (collection) => {
   const validTags = ['post', 'pathItem'];
   const eleventyPosts = collection
-    .getAll()
+    .getFilteredByGlob('**/*.md')
```

Comment:
This seems unrelated but probably fine. I presume this fixes the issue that we're e.g. indexing our empty partial.

Reply:
yeah I added this because I noticed we were running all sorts of stuff like sitemap.xml through liveposts and that seemed...bad.